bundles/org.openhab.binding.shelly/README.md

```diff
@@ -1551,6 +1560,14 @@ See notes on discovery of Shelly BLU devices above.
 | battery | batteryLevel  | Number   | yes       | Battery Level in %                                      |
 |         | lowBattery    | Switch   | yes       | Low battery alert (< 20%)                               |
 
+## Shelly BLU Gateway (thing-type: shellyblugw)
+
+| Group   | Channel       | Type     | read-only | Description                                             |
+| ------- | ------------- | -------- | --------- | ------------------------------------------------------- |
+
+There no no additional channels beside the device group.
+
+
```

Comment:
I guess there is no reason to include an empty table?
```suggestion
There are no additional channels beside the device group.

```

Reply:
removed

Reply:
Sorry, I suggested also to correct a type, but have to admit it was well hidden.

Reply:
:-)